# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libchipcard/64bit.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur libchipcard-0.9.1/src/libchipcard/cards/hbcicard.cpp libchipcard-0.9.1-t2/src/libchipcard/cards/hbcicard.cpp
--- libchipcard-0.9.1/src/libchipcard/cards/hbcicard.cpp	2003-05-07 22:14:59.000000000 +0200
+++ libchipcard-0.9.1-t2/src/libchipcard/cards/hbcicard.cpp	2006-01-29 00:29:49.000000000 +0100
@@ -684,8 +684,8 @@
   case k_HBCICARD_TYPE_1:
     return _hash2mac1(hash,mac);
   default:
-    DBG_INFO("LIBCHIPCARD: unknown chip card type (%8x) (%d).",
-	     (unsigned int)this,
+    DBG_INFO("LIBCHIPCARD: unknown chip card type (%8x) (%ld).",
+	     (unsigned long)this,
 	    _type);
     return false;
   }
